src/Resources/contao/dca/tl_module.php

```diff
@@ -16,266 +16,29 @@
-    'reference' => &$GLOBALS['TL_LANG']['tl_module']['dk_mmenuPageDimOptions'],
-    'eval' => ['tl_class' => 'w50', 'includeBlankOption' => true],
-    'sql' => "varchar(16) NOT NULL default ''",
-];
-
-$GLOBALS['TL_DCA']['tl_module']['fields']['dk_mmenuMenuEffects'] = [
-    'label' => &$GLOBALS['TL_LANG']['tl_module']['dk_mmenuMenuEffects'],
-    'exclude' => true,
-    'inputType' => 'select',
-    'options' => ['fade', 'slide', 'zoom'],
-    'eval' => ['tl_class' => 'w50', 'includeBlankOption' => true],
-    'sql' => "varchar(16) NOT NULL default ''",
-];
-
-$GLOBALS['TL_DCA']['tl_module']['fields']['dk_mmenuPanelEffects'] = [
-    'label' => &$GLOBALS['TL_LANG']['tl_module']['dk_mmenuPanelEffects'],
-    'exclude' => true,
-    'inputType' => 'select',
-    'options' => ['none', 'slide-0', 'slide-100', 'slide-up', 'zoom'],
-    'eval' => ['tl_class' => 'w50', 'includeBlankOption' => true],
-    'sql' => "varchar(16) NOT NULL default ''",
-];
-
-$GLOBALS['TL_DCA']['tl_module']['fields']['dk_mmenuListEffects'] = [
-    'label' => &$GLOBALS['TL_LANG']['tl_module']['dk_mmenuListEffects'],
-    'exclude' => true,
-    'inputType' => 'select',
-    'options' => ['drop', 'fade', 'slide'],
-    'eval' => ['tl_class' => 'w50', 'includeBlankOption' => true],
-    'sql' => "varchar(16) NOT NULL default ''",
-];
-
-$GLOBALS['TL_DCA']['tl_module']['fields']['dk_mmenuDragOpenEnable'] = [
-    'label' => &$GLOBALS['TL_LANG']['tl_module']['dk_mmenuDragOpenEnable'],
-    'exclude' => true,
-    'inputType' => 'checkbox',
-    'eval' => ['submitOnChange' => true, 'tl_class' => 'clr'],
-    'sql' => "char(1) NOT NULL default ''",
-];
-
-$GLOBALS['TL_DCA']['tl_module']['fields']['dk_mmenuDragOpenThreshold'] = [
-    'label' => &$GLOBALS['TL_LANG']['tl_module']['dk_mmenuDragOpenThreshold'],
-    'exclude' => true,
-    'inputType' => 'text',
-    'eval' => ['maxlength' => 3, 'rgxp' => 'digit', 'tl_class' => 'w50'],
-    'sql' => "smallint(5) NOT NULL default '50'",
-];
-
-$GLOBALS['TL_DCA']['tl_module']['fields']['dk_mmenuDragOpenMaxStartPos'] = [
-    'label' => &$GLOBALS['TL_LANG']['tl_module']['dk_mmenuDragOpenMaxStartPos'],
-    'exclude' => true,
-    'inputType' => 'text',
-    'eval' => ['maxlength' => 3, 'rgxp' => 'digit', 'tl_class' => 'w50'],
-    'sql' => "smallint(5) NOT NULL default '100'",
-];
-
-$GLOBALS['TL_DCA']['tl_module']['fields']['dk_mmenuPolyfillEnable'] = [
-    'label' => &$GLOBALS['TL_LANG']['tl_module']['dk_mmenuPolyfillEnable'],
-    'exclude' => true,
-    'inputType' => 'checkbox',
-    'eval' => ['tl_class' => 'clr'],
-    'sql' => "char(1) NOT NULL default ''",
-];
-
-$GLOBALS['TL_DCA']['tl_module']['fields']['dk_mmenuOnClickClose'] = [
-    'label' => &$GLOBALS['TL_LANG']['tl_module']['dk_mmenuOnClickClose'],
-    'exclude' => true,
-    'inputType' => 'checkbox',
-    'eval' => ['tl_class' => 'clr w50'],
-    'sql' => "char(1) NOT NULL default ''",
-];
-
 $GLOBALS['TL_DCA']['tl_module']['fields']['dk_mmenuJsTpl'] = [
```

Comment:
I decided to keep the dk_mmenuJsTpl in the module config - could also be moved to the config table but then the config loading cannot be handled in the `MmenuHelper`